### DISCUSSION TITLE
[疑惑] -t 没有按照预期工作

### DISCUSSION THREAD

**Discussion by luelueking** [projectdiscovery/nuclei#5466] [Q&A]
为什么会显示File Not Found ， -t不是指定模板文件的吗
<img width="1621" alt="截屏2024-07-30 12 27 49" src="https://github.com/user-attachments/assets/fa8e5c59-0b02-4a81-93c6-74991c9a6351">

**Comment by ehsandeep** [comment]
@luelueking `.yaml` valid extension for nuclei template.
